internal/policy/src/main/resources/policy/checkstyle.xml

```diff
@@ -70,6 +72,10 @@
             <property name="sortStaticImportsAlphabetically" value="true"/>
         </module>
         <!--<module name="InnerAssignment"/>-->
+        <module name="Indentation">
```

Comment:
isn't that just repeating the defaults? According to https://checkstyle.sourceforge.io/config_misc.html#Indentation, there are six settings. Why are these two special if we don't change the defaults?

Reply:
Yes, these are the defaults. I find some value in spelling defaults out nevertheless, as not everyone will be familiar what they are. But I guess the same could be true in so many other places ... I'll remove them!